apps/web/src/services/auth/components/SignInContainer.vue

```diff
@@ -1,26 +1,30 @@
 <script lang="ts" setup>
 import { useWindowSize } from '@vueuse/core/index';
 import { computed, reactive } from 'vue';
+import { useRoute } from 'vue-router/composables';
 
 import { screens } from '@cloudforet/mirinae';
 
 import ConsoleLogo from '@/services/auth/components/ConsoleLogo.vue';
 import SignInLeftContainer from '@/services/auth/components/SignInLeftContainer.vue';
 
-
+const route = useRoute();
 const { width } = useWindowSize();
 
 const state = reactive({
     isMobileSize: computed<boolean>(() => width.value < screens.mobile.max),
+    isCentered: computed<boolean>(() => route.meta?.isCentered),
```

Comment:
nitpick: If `route.meta?.isCentered` could be `undefined`, how about handling it like this?
```
!!route.meta?.isCentered
```